modules/core/src/main/java/org/locationtech/jts/io/WKBWriter.java

```diff
@@ -211,6 +212,7 @@ private static char toHexDigit(int n)
     return (char) ('A' + (n - 10));
   }
 
+  private EnumSet<Ordinate> outputOrdinates;
```

Comment:
I think the use of a set here is over-designed.  There's really only 3 cases that need to be handled: Z, M and ZM.  This could be done more simply with a single Enum or even a group of `static int` constants.

Reply:
I borrowed the concept of this EnumSet from the WKTWriter. I think both should work the same way. So I would suggest to refactor WKTWriter and WKBWriter together in a new issue...

Reply:
Why not design for more measure (if needed) over time? I guess because it is a standard and just has those three use-cases...